pkg/terraform/exec/plugins/alibabacloud.go

```diff
@@ -0,0 +1,17 @@
+package plugins
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/plugin"
+)
+
+func init() {
+	// TODO AlibabaCloud: A later PR
+	// "github.com/terraform-providers/terraform-provider-alicloud/alicloud"
```

Comment:
This is something that we need to update. I attempted to add https://github.com/aliyun/terraform-provider-alicloud/tree/master/alicloud and run with the v1.129.0. It looks like we need to update to the latest APIs.
```
# github.com/aliyun/terraform-provider-alicloud/alicloud
vendor/github.com/aliyun/terraform-provider-alicloud/alicloud/resource_alicloud_cs_kubernetes_autoscaler.go:450:73: not enough arguments in call to clientSet.AppsV1().Deployments(defaultAutoscalerNamespace).Delete
	have (string, *"k8s.io/apimachinery/pkg/apis/meta/v1".DeleteOptions)
	want (context.Context, string, "k8s.io/apimachinery/pkg/apis/meta/v1".DeleteOptions)
vendor/github.com/aliyun/terraform-provider-alicloud/alicloud/resource_alicloud_cs_kubernetes_autoscaler.go:461:79: not enough arguments in call to clientSet.AppsV1().Deployments(defaultAutoscalerNamespace).Get
	have (string, "k8s.io/apimachinery/pkg/apis/meta/v1".GetOptions)
	want (context.Context, string, "k8s.io/apimachinery/pkg/apis/meta/v1".GetOptions)
vendor/github.com/aliyun/terraform-provider-alicloud/alicloud/resource_alicloud_cs_kubernetes_autoscaler.go:538:79: not enough arguments in call to clientSet.AppsV1().Deployments(defaultAutoscalerNamespace).Create
	have (*"k8s.io/api/apps/v1".Deployment)
	want (context.Context, *"k8s.io/api/apps/v1".Deployment, "k8s.io/apimachinery/pkg/apis/meta/v1".CreateOptions)
vendor/github.com/aliyun/terraform-provider-alicloud/alicloud/resource_alicloud_cs_kubernetes_autoscaler.go:549:78: not enough arguments in call to clientSet.AppsV1().Deployments(defaultAutoscalerNamespace).Update
	have (*"k8s.io/api/apps/v1".Deployment)
	want (context.Context, *"k8s.io/api/apps/v1".Deployment, "k8s.io/apimachinery/pkg/apis/meta/v1".UpdateOptions)
vendor/github.com/aliyun/terraform-provider-alicloud/alicloud/resource_alicloud_cs_kubernetes_autoscaler.go:634:81: not enough arguments in call to clientSet.CoreV1().ConfigMaps(defaultAutoscalerNamespace).Get
	have (string, "k8s.io/apimachinery/pkg/apis/meta/v1".GetOptions)
	want (context.Context, string, "k8s.io/apimachinery/pkg/apis/meta/v1".GetOptions)
vendor/github.com/aliyun/terraform-provider-alicloud/alicloud/resource_alicloud_cs_kubernetes_autoscaler.go:637:77: not enough arguments in call to clientSet.CoreV1().ConfigMaps(defaultAutoscalerNamespace).Create
	have (*"k8s.io/api/core/v1".ConfigMap)
	want (context.Context, *"k8s.io/api/core/v1".ConfigMap, "k8s.io/apimachinery/pkg/apis/meta/v1".CreateOptions)
vendor/github.com/aliyun/terraform-provider-alicloud/alicloud/resource_alicloud_cs_kubernetes_autoscaler.go:649:75: not enough arguments in call to clientSet.CoreV1().ConfigMaps(defaultAutoscalerNamespace).Update
	have (*"k8s.io/api/core/v1".ConfigMap)
	want (context.Context, *"k8s.io/api/core/v1".ConfigMap, "k8s.io/apimachinery/pkg/apis/meta/v1".UpdateOptions)
```
It appears a `context.Context` object was added.  Can we get this updated?

Reply:
The PR is under review, we will push this update as soon as possible.

Reply:
@kwoodson This aliyun/terraform-provider-alicloud repository has been updated, and I have updated the alibabalcoud provider plugin in Installer. I think you can continue to review and test. At present, I need to provide the mirror ID after testing.